test/Common.FunctionalTests/Inprocess/LoggingTests.cs

```diff
@@ -77,6 +71,30 @@ public async Task InvalidFilePathForLogs_ServerStillRuns()
             await Helpers.AssertStarts(deploymentResult, "HelloWorld");
         }
 
+        [ConditionalFact]
+        public async Task OnlyOneFileCreated()
+        {
+            var deploymentParameters = _fixture.GetBaseDeploymentParameters(publish: true);
+
+            deploymentParameters.EnableLogging(_logFolderPath);
+
+            var deploymentResult = await DeployAsync(deploymentParameters);
+
+            await Helpers.AssertStarts(deploymentResult, "CheckLogFile");
+
+            StopServer();
+
+            var files = Directory.GetFiles(_logFolderPath);
+            Assert.Single(files);
+
+            var starttime = deploymentResult.HostProcess.StartTime.ToUniversalTime();
+            Assert.Equal(Path.Combine(_logFolderPath, $"std_{starttime.Year}{starttime.Month.ToString().PadLeft(2, '0')}" +
+                $"{starttime.Day.ToString().PadLeft(2, '0')}{starttime.Hour.ToString().PadLeft(2, '0')}" +
```

Comment:
https://docs.microsoft.com/en-us/dotnet/standard/base-types/how-to-pad-a-number-with-leading-zeros

Reply:
Yeah that's about 3 million times better than what I did.